token-endpoint-richiesta-token/response-1.rst

```diff
@@ -14,7 +14,12 @@ OAuth 2.0 - Draft 03, paragrafo 3.2.1, “JWT Bearer Tokens”.
 
 .. code-block:: json
 
- { 
+ HTTP/1.1 200 OK
```

Comment:
This response in the SPID OIDC Profile is in JWT format and the example here is the payload in it.
This mean that OIDC SPID MUST adopt the token response as JWS